app/queues/notifications/record-cert-expiration-data.interface.ts

```diff
@@ -0,0 +1,14 @@
+// data we expect to get back from database when record/certificate validation
+interface User {
+  username: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  createdAt: Date;
+  updatedAt: Date;
+}
+export interface ExpirationStatusPayload {
+  user: User;
```

Comment:
When you move this into the other file, you can also import the `User` type from Prisma, and use that vs. redefining it yourself.